docs/content/en/references/apps_v1alpha1_types.html

```diff
@@ -1616,7 +1616,8 @@ <h3 id="apps.kurator.dev/v1alpha1.Metric">Metric
 </td>
 <td>
 <p>Name of the metric.
-Currently supported metric are <code>request-success-rate</code> and <code>request-duration</code>.</p>
+Currently supported metric are <code>request-success-rate</code> and <code>request-duration</code>.
+When Name is not <code>request-success-rate</code> or <code>request-duration</code>, you need to define the metric rule in CustomMetric.</p>
```

Comment:
Even if request-success-tare and request-duration are not available, you can use the metrics that come with the gateway without customising them.
Both scenarios should be made clear